pom.xml

```diff
@@ -135,6 +136,7 @@
         <jersey.version>2.35</jersey.version>
         <jetty.version>9.4.41.v20210516</jetty.version>
         <jline.version>0.9.94</jline.version>
+        <junit.version>4.13.2</junit.version>
```

Comment:
for these two properties, they are only used in kyuubi-rest-client module.

Let us move to kyuubi-rest-client module.